src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2170,11 +2170,11 @@ def aks_agentpool_add(
     tags=None,
     node_taints=None,
     node_osdisk_type=None,
-    node_osdisk_size=0,
+    node_osdisk_size=None,
     max_surge=None,
     mode=CONST_NODEPOOL_MODE_USER,
     scale_down_mode=CONST_SCALE_DOWN_MODE_DELETE,
-    max_pods=0,
+    max_pods=None,
```

Comment:
Let me check if this PR will not only change the default value in `az aks create/update` command, but also other command like `az aks nodepool add`?
If so, please add this part of info to the title and description of this PR.

Reply:
Good catch, the change is applied to command `az aks create` and `az aks nodepool add`

Reply:
updated PR title